plugins/services/src/js/components/dsl/FuzzyTextDSLSection.js

```diff
@@ -40,13 +44,37 @@ class FuzzyTextDSLSection extends React.Component {
   }
 
   handleTextChange(event) {
+    const { onChange, expression } = this.props;
     const { target } = event;
     event.stopPropagation();
+
+    const value = target.value;
```

Comment:
Looks like we could just go with `const { target: { value } } = event;`

Reply:
Correct.